tests/ImportCommentsPreserved/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -0,0 +1,178 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`import-comments-babel-pulling-single-line-bug.ts - typescript-verify > import-comments-babel-pulling-single-line-bug.ts 1`] = `
+
+import B from "b";
+// Trailing comment (not supposed to be on same line)
+// Bottom of imports single-line comment
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+import B from "b";
+// Trailing comment (not supposed to be on same line)
+
+// Bottom of imports single-line comment
+
+`;
+
+exports[`import-comments-document-top-test1.ts - typescript-verify > import-comments-document-top-test1.ts 1`] = `
+// This is part of multiple-lines at document-top
+// Second line followed by a gap
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// This is part of multiple-lines at document-top
+// Second line followed by a gap
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-document-top-test2.ts - typescript-verify > import-comments-document-top-test2.ts 1`] = `
+/**
+ * This is part of multiple-lines at document-top
+ * Second line of text, comment is followed by a gap (gap gets eaten)
+ */
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * This is part of multiple-lines at document-top
+ * Second line of text, comment is followed by a gap (gap gets eaten)
+ */
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-document-top-test3.ts - typescript-verify > import-comments-document-top-test3.ts 1`] = `
+/**
+ * This comment is before the whole block,
+ * it is not attached to the following import.
+ */
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * This comment is before the whole block,
+ * it is not attached to the following import.
+ */
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-outer-comments-block-comments.ts - typescript-verify > import-comments-outer-comments-block-comments.ts 1`] = `
+// Loose leading comment before imports (should not be dragged down with B)
+
+/* Leading comment before B */
+import B from "b"; /* Trailing comment on same-line as B (babel doesn't like CommentBlocks on same-line as anything else, so it will be shifted to a clean line) */
```

Comment:
Hmmm, it would be great to keep this together with the import.  Where its ending up feels a little broken.